### ISSUE TITLE
JS error when running the app as e.g. iPhone, Galaxy or Responsive (Chrome / F12)

### ISSUE THREAD

**Issue by FransVerschoor** [AuraQ/AutoCompleteForMendix#25]
Hi Iain,

The following error only occurs regularly when running the app as e.g. iPhone, Galaxy or Responsive (Chrome / F12):
```
Uncaught TypeError: ((jQuery.event.special[handleObj.origType] || (intermediate value)).handle || handleObj.handler).apply is not a function
    at HTMLDocument.dispatch (jquery-1.11.2.js:4665)
    at HTMLDocument.elemData.handle (jquery-1.11.2.js:4333)
```
You can use the project I send you before and you modified to reproduce.

Thanks,

Frans

**Comment by lindski** [comment]
Hi Frans

There was a dodgy line in the widget that was binding to an event with no event handler defined which I have now removed.

You can take an updated copy of the widget from the test project if you want to try it out.

Cheers

Iain

**Comment by lindski** [comment]
Fixed in 3.2.0
